test/bigtest/interactors/pci-agreements.js

```diff
@@ -0,0 +1,24 @@
+import {
+  interactor,
+  text,
+} from '@bigtest/interactor';
+
+import MultiColumnListInteractor from '@folio/stripes-components/lib/MultiColumnList/tests/interactor';
+
+export default @interactor class PCIAgreementsInteractor {
+  agreementsList = new MultiColumnListInteractor('#pciagreements-agreements');
+  relatedAgreementsList = new MultiColumnListInteractor('#pcirelatedagreements-agreements');
```

Comment:
`id`s should generally be `kebab-cased`. Sometimes that was awkward with accordion sections when we had to manually manage the open/close state so we used `camelCasing` but I don't think we need to do that anymore with AccordionStatus 🤔